pkg/app/app.go

```diff
@@ -102,10 +118,14 @@ func (a *App) MainLoop(
 		a.FpsCounter.Update(deltaTime, 1)
 		oldTime = newTime
 
-		a.Context.Clear()
+		a.Clear()
 		render()
+
 		a.FpsCounterText.SetText(fmt.Sprintf("%v", a.FpsCounter.FPS()))
-		a.Context.EnqueueForDrawing(a.FpsCounterText)
+		a.UIContext.EnqueueForDrawing(a.FpsCounterText)
```

Comment:
Can we move the uicontext drawing after the main context is rendered? I know it doesn't change much but it give better the idea that the UI is on top of everything else

Reply:
yeah, that was me moving the code around trying to debug the glitch